module/migration.mjs

```diff
@@ -267,6 +267,8 @@ export const migrateActorData = function(actor, migrationData) {
     const itemData = i instanceof CONFIG.Item.documentClass ? i.toObject() : i;
     let itemUpdate = migrateItemData(itemData, migrationData);
 
+    if ( itemData.type === "background" ) updateData["system.details.background"] = itemData._id;
```

Comment:
Let's make sure this is idempotent and we are not unnecessarily performing an update.

```suggestion
    if ( (itemData.type === "background") && (actor.system?.details?.background !== itemData._id) ) {
      updateData["system.details.background"] = itemData._id;
    }
```